src/ocamlorg_frontend/pages/packages.eml

```diff
@@ -6,7 +6,15 @@ Layout.render
 ~description:"Discover thousands of community packages and browse their documentation."
 ~canonical:Url.packages
 ~active_top_nav_item:Header.Packages @@
-<div class="intro-section-simple">
+
+let search_dropdown () =
+  <div id="packages-search-results" aria-live="polite" ></div>
+  <a class="flex py-2 px-4 gap-4 hover:bg-primary-100 font-normal rounded-md text-primary-600" href="<%s Url.api %>">
```

Comment:
```suggestion
  <a class="flex py-2 px-2 mx-2 gap-4 hover:bg-primary-100 font-normal rounded-md text-primary-600" href="<%s Url.api %>">
```

there's still something about the alignment, I think this makes the StdLib link align better with the rest when hovered.